modeling/dataset/samplers/next_item_prediction.py

```diff
@@ -268,43 +301,55 @@ def __getitem__(self, index):
             labels = [1] + [0] * len(negatives)
 
             result.update({
-                'item.%s.ids'%(domain): item_sequence,
-                'item.%s.length'%(domain): len(item_sequence),
+                'item.{}.ids'.format(domain): item_sequence,
+                'item.{}.length'.format(domain): len(item_sequence),
 
-                'candidates.%s.ids'%(domain): candidates,
-                'candidates.%s.length'%(domain): len(candidates),
+                'candidates.{}.ids'.format(domain): candidates,
+                'candidates.{}.length'.format(domain): len(candidates),
 
-                'labels.%s.ids'%(domain): labels,
-                'labels.%s.length'%(domain): len(labels)
+                'labels.{}.ids'.format(domain): labels,
+                'labels.{}.length'.format(domain): len(labels)
             })
 
         return result
 
 
 class MultiDomainNextItemPredictionEvalSampler(MultiDomainEvalSampler, config_name='multi_domain_next_item_prediction'):
 
-    def __init__(self, dataset, num_users, num_items, target_domain, other_domains):
+    def __init__(
+            self, 
+            dataset, 
+            num_users, 
+            num_items, 
+            target_domain, 
+            other_domains
+        ):
+        
         super().__init__()
```

Comment:
You should pass some params in the `super().__init__()` based on its definition in the `MultiDomainEvalSampler`. 

```def __init__(self, dataset, num_users, num_items):```